<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>my_widget</class>
 <widget class="QWidget" name="my_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="widget_layout">
     <property name="spacing">
      <number>5</number>
     </property>
     <item row="0" column="0">
      <widget class="QDateEdit" name="pay_date">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QDateEdit {
    border-radius: 10px;
    padding: 0 8px;
	background-color: rgb(226, 226, 226);
	color:black;
	border: 1px solid black;
}

QDateEdit::drop-down {
    border-left-width: 1px;
    border-left-color: black;
    border-left-style: solid;
    width: 20px;
    height: 35px;
    border-top-right-radius: 9px;
    border-bottom-right-radius: 9px;
}

QDateEdit QCalendarWidget {
    color: black;
}

QDateEdit QCalendarWidget QAbstractItemView {
    background-color: white;
    color: black;
}

QDateEdit QCalendarWidget QToolButton {
    color: black;
}

QDateEdit QCalendarWidget QToolButton QMenu {
    color: black;
}</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="date">
        <date>
         <year>2019</year>
         <month>1</month>
         <day>11</day>
        </date>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="5">
      <widget class="QTableWidget" name="table">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QTableWidget::item:selected {
	background-color: rgb(175, 97, 33);
	color: rgb(255, 255, 255);
}

QTableWidget {
    border-radius: 10px;
    padding: 0 8px;
	background-color: rgb(226, 226, 226);
	color:black;
	color: rgb(0, 0, 0);
	gridline-color: rgb(0, 0, 0);
	border: 1px solid black;
}

QTableCornerButton::section { 
	background-color: rgb(226, 226, 226); 
}
</string>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="horizontalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="rowCount">
        <number>0</number>
       </property>
       <column>
        <property name="text">
         <string>Date</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Sum</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Recalculation</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="pay_sum">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">    border-radius: 10px;
    padding: 0 8px;
	background-color: rgb(226, 226, 226);
	color:black;
	border: 1px solid black;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Payment sum</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QPushButton" name="clean_table">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
border-radius: 10px;
background-color: rgb(226, 226, 226);
color:black;
border: 1px solid black;
}

QPushButton:pressed {
background-color: rgb(170, 44, 33);
}</string>
       </property>
       <property name="text">
        <string>×</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="pay_type">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox {
    border-radius: 10px;
    padding-right: 20px;
    padding: 0 8px;
    background-color: rgb(226, 226, 226);
    color: black;
    border: 1px solid black;
}

QComboBox::drop-down {
    border-left-width: 1px;
    border-left-color: black;
    border-left-style: solid;
    width: 20px;
    height: 35px;
    border-top-right-radius: 9px;
    border-bottom-right-radius: 9px;
}

QComboBox::down-arrow {
    width: 16px;
    height: 16px;
}
</string>
       </property>
       <item>
        <property name="text">
         <string>Reduce term</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Reduce payment</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="3" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QPushButton" name="add_to_table">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
border-radius: 10px;
background-color: rgb(226, 226, 226);
color:black;
border: 1px solid black;
}

QPushButton:pressed {
background-color: rgb(175, 97, 33);
}</string>
       </property>
       <property name="text">
        <string>✓</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
